django_project/realtime/tasks/ash.py

```diff
@@ -7,23 +7,48 @@
 from tempfile import mkdtemp
 
 import pytz
+
+from django.core.files import File
+
 from celery.result import AsyncResult
 
 from core.celery_app import app
+
 from realtime.app_settings import LOGGER_NAME, REALTIME_HAZARD_DROP
-from realtime.models.ash import Ash
+from realtime.models.ash import Ash, AshReport
 from realtime.tasks.realtime.ash import process_ash
 from realtime.tasks.realtime.celery_app import app as realtime_app
+from realtime.tasks.headless.celery_app import app as headless_app
+
+from realtime.tasks.headless.inasafe_wrapper import (
+    run_multi_exposure_analysis, generate_report)
 
 __author__ = 'Rizky Maulana Nugraha <lana.pcfre@gmail.com>'
 __date__ = '20/7/17'
 
 
 LOGGER = logging.getLogger(LOGGER_NAME)
 
+ASH_EXPOSURES = [
```

Comment:
We need this to be easily customizable by user somehow later.

Options are:
- Save the options in the database
- Save the options in environment variables

In either case, I decided to move these default options into app_settings